app/views/content/is-teaching-right-for-me/computing.md

```diff
@@ -0,0 +1,22 @@
+---
+title: How to become a computing teacher
```

Comment:
With the current header design on these pages, I think this title/h1 is probably too long :-( 

Would we be better off with:
* `title` - How to become a computing teacher
* `h1` - Become a computing teacher